winterfell/webapp/src/js/noangular/signin.js

```diff
@@ -36,7 +38,30 @@ function displayError(msg) {
   $('.error-msg').text(msg);
 }
 
+function adjustTextsForLogin() {
```

Comment:
This feels a little weird to me having one page and then editing the DOM to make it look like either signup or login. I'm afraid it might make it hard to change either of these pages. I think it might be a lot nicer to use jade/pug's inheritance features:

https://pugjs.org/language/inheritance.html

We could have one base signin page, and then have two different blocks with either the fields for creating an account or the fields for logging in